modules/light-clients/08-wasm/client/cli/tx.go

```diff
@@ -82,3 +82,49 @@ func newSubmitStoreCodeProposalCmd() *cobra.Command {
 
 	return cmd
 }
+
+func newMigrateContractCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "migrate-contract [client-id] [checksum] [migrate-msg]",
```

Comment:
I renamed `new-code-msg` to `migrate-msg`, since this is a JSON-encoded string that is passed to the contract that is being migrated.